mathematica/integrator_plots.cpp

```diff
@@ -200,6 +303,12 @@ class WorkErrorGraphGenerator {
       v_errors_[i].resize(integrations_per_integrator_);
       e_errors_[i].resize(integrations_per_integrator_);
       evaluations_[i].resize(integrations_per_integrator_);
+      for (int j = 0; j < integrations_per_integrator_; ++j) {
```

Comment:
Could we try to use `std::int64_t` instead of `int`?  (Everywhere in this PR.)

Reply:
OK, but then at some point we need to do a general cleanup of the int-by-default indices in this codebase.